experimental/dds/tree/src/interface-prototype/Sequence.ts

```diff
@@ -0,0 +1,16 @@
+// Sequence generic
+// TODO: are sequences mutable? Are locations usable across mutations? What about Iterators?
+
+export interface Sequence<Element, Place = Element | undefined> {
+	[Symbol.iterator](): SequenceIterator<Element, Place>;
+	iteratorFromEnd(): SequenceIterator<Element, Place>;
+	areInOrder(first: Element, second: Element): boolean;
+	readonly length: number;
+	indexOf(element: Element): number;
+	elementAtIndex(index: number): Element;
```

Comment:
Consider just 'at(index) to match [TC39 propasal](https://tc39.es/proposal-relative-indexing-method/).

Reply:
I like that idea.

Reply:
done